decidim-assemblies/app/serializers/decidim/assemblies/assembly_importer.rb

```diff
@@ -140,6 +141,18 @@ def import_components(components)
         importer.import(components, @user)
       end
 
+      def import_hero_image(attribute)
```

Comment:
In order to keep things DRY, could you please use the uploader's `remote_url=` setter which already does most of the things you're doing here:
https://github.com/decidim/decidim/blob/5da008a83e9d3ed0003799eedc648a34d7176d2a/decidim-core/app/uploaders/decidim/application_uploader.rb#L64-L71

Example:
https://github.com/decidim/decidim/blob/5da008a83e9d3ed0003799eedc648a34d7176d2a/decidim-proposals/app/services/decidim/proposals/proposal_builder.rb#L120

This would apply to the assembly importer as well as the changes in the process importer.

Reply:
Thank you for the suggestion, it's great!